src/platform/windows/startup-issues.md

```diff
@@ -4,13 +4,13 @@
 
 ## No error, but app does not appear
 
-Recently there have been some reports of Anki failing to appear, without
-any error message showing up. If you run into this situation, you can either:
+If you start Anki and it fails to appear, without
+any error message, you can try:
```

Comment:
```suggestion
any error message, you can try the following:
```

better to complete the sentence maybe, and it helps keep the gerunds out, which is confusing for some ESLs and we also normally don't use them in Manual.